cscs-checks/microbenchmarks/cpu/likwid/likwid.py

```diff
@@ -15,7 +15,8 @@ def __init__(self):
 
         self.executable = 'likwid-bench'
 
-        self.num_tasks = 0
+        self.num_tasks = 4
```

Comment:
@ChristopherBignamini, is there any specific reason why you are selecting the number 4?

Reply:
As discussed, I will set it to 1.